docs/best-practices/gitops-with-kruise.md

```diff
@@ -32,97 +34,14 @@ $ helm upgrade kruise openkruise/kruise --set featureGates="TemplateNoDefaults=t
 ```
 
 ### Declarative Applications in Git Repository
-1. OpenKruise provides a guestbook application [demo](https://github.com/openkruise/samples).
-**It consists of guestbook, redis, through openKruise CloneSet to complete the deployment of the application.** As follows:
+This article provides a helloworld http service [demo](https://github.com/zmberg/samples/tree/hello_world/helloworld), It contains Helm Charts, as follows:
 
-![guestbook](/img/docs/best-practices/guestbook.png)
+![gitops helloworld](/img/docs/best-practices/gitops_helloworld.png)
 
```

Comment:
plz keep the Configuring Argo Application section, so that the document is more complete